apps/web/components/RotatingText.tsx

```
@@ -0,0 +1,232 @@
+  ) => {
+    const [currentTextIndex, setCurrentTextIndex] = useState<number>(0);
+
+    const splitIntoCharacters = (text: string): string[] => {
+      if (typeof Intl !== 'undefined' && Intl.Segmenter) {
+        const segmenter = new Intl.Segmenter('en', { granularity: 'grapheme' });
+        return Array.from(segmenter.segment(text), segment => segment.segment);
+      }
+      return Array.from(text);
+    };
+
+    const elements = useMemo(() => {
```

Biome (2.1.2)

[error] This hook does not specify its dependency on splitIntoCharacters.

This dependency is being used here, but is not specified in the hook dependency list.

Unsafe fix: Add the missing dependency to the list.


(lint/correctness/useExhaustiveDependencies)